admin-api-server/src/analytics/service/analytics.service.ts

```diff
@@ -117,6 +119,39 @@ export class AnalyticsService {
     });
   }
 
+  async getUsers(params: PaginationParams) {
+    const qryRes = await this.storeRepl.query(
+      `SELECT
+        id,
+        address,
+        email,
+        consent,
+        created_at
+        FROM marketing 
+        ORDER BY "${params.orderBy}" ${params.orderDirection}
+        OFFSET ${params.pageOffset}
+        LIMIT ${params.pageSize}`,
+    );
+
+    if (qryRes.rowCount === 0) {
+      return { data: [], count: 0 };
```

Comment:
I don't think this special `if` case is necessary? If rowCount is 0, then `qryRes.rows` will be `[]`, and a `qryRes.rows.map` will then result in `[]`. Therefore, removing this `if` will result in the same return value.

Reply:
Yes, removed it